ui/StatusQ/src/StatusQ/Controls/StatusInput.qml

```diff
@@ -515,13 +523,33 @@ Control {
             onEditingFinished: {
                 root.editingFinished()
             }
+
+            // use the default padding values from StatusBaseInput, but forward any explicitly set values from outside
+            Binding on topPadding {
```

Comment:
Wouldn't it be cleaner to remove the margin from StatusBaseInput and use it's default value as a default value for StatusInput itself?

Reply:
Yeah, I can give it a try

Reply:
Yeah but still the same problem; how do you set it from outside _while_ retaining those defaults?

Reply:
Hmm and the meaning (when `StatusBaseInput` is used inside `StatusInput` is completely different)